fog/view/server/src/router_request_handler.rs

```diff
@@ -67,14 +70,19 @@ pub async fn handle_request<E>(
     shard_clients: Vec<Arc<FogViewStoreApiClient>>,
     enclave: E,
     logger: Logger,
+    tracer: &BoxedTracer,
```

Comment:
Whats the reason for not getting the tracer using the `tracer!()` macro inside this method?

Reply:
I don't have a great reason, other than I thought it would be less wasteful to pass a reference to a pre-constructed tracer?

Reply:
However it looks like its being created inside the loop so I am not sure if that saves anything.